src/olympia/amo/tests/__init__.py

```diff
@@ -943,6 +944,7 @@ def version_factory(file_kw=None, **kw):
         file_factory(version=ver, **file_kw)
     if promotion_approved:
         kw['addon'].promotedaddon.approve_for_version(version=ver)
+    kw['addon'].update(status=kw['addon'].status)  # forcing an update
```

Comment:
`addon_factory()` takes great care to set up `save` calls in a very specific order, so I'm not sure this is a good idea to force an update when creating a version

Reply:
This was something I added as a fix for one of the tests - I can't remember which now, or if it was actually necessary in th end